sales.html

```diff
@@ -7,13 +7,30 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
     <body>
-        <h1>Salmon Cookies: Hourly Sales Projections</h1>
-        <table>
-            <caption>Cookies Needed By Location Each Day</caption>
-            <thead></thead>
-            <tbody></tbody>
-            <tfoot></tfoot>
-        </table>
+        <article>
```

Comment:
I like the use of semantic tags to try and separate the content, but I'm not sure `<article>` is the correct one. [The MDN documentation for it mentions that it communicates an element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article):
> is intended to be independently distributable or reusable

The table could probably be argued to possibly be an `article`, but I don't know about the `form`, given that it's tied pretty closely to the `table`.

Maybe we could remove the `<article>` on line 19 and convert the `<h2>` into a [`<legend> element`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend) that's a child of the `form`?

Reply:
Both of these article sections are being used to display flex, and legends seem better suited for a section of a form that needs a label, not for the whole form itself.